### ISSUE TITLE
exclude chars from uD800 to uDEFF in DefaultCharacterArbitrary

### ISSUE THREAD

**Issue by michaelmosmann** [jlink/jqwik#64]
see https://en.wikipedia.org/wiki/UTF-16#U+D800_to_U+DFFF

## Testing Problem

If a String generated by jqwik is serialized as byte stream and read back (f.I. persist to database) encoding errors will lead to different string value.

## Suggested Solution

change method in DefaultCharacterArbitrary:
```
	static boolean isNoncharacter(int codepoint) {
		if (codepoint >= 0xfdd0 && codepoint <= 0xfdef)
			return true;
		if (codepoint >= 0xd800 && codepoint <= 0xdfff)
			return true;
		return codepoint == 0xfffe || codepoint == 0xffff;
	}
```

**Comment by jlink** [comment]
I applied the suggested change. 

Is available in 1.2.0-SNAPSHOT release.

**Comment by michaelmosmann** [comment]
thanks. :)
